layer-publisher/tests/unit/layer-publisher.test.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Test layer publisher
+ *
+ * @group unit/layer-publisher
+ */
+
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import * as LayerPublisher from '../../src/layer-publisher-stack';
+
+test('Layer  Created', () => {
+  const app = new cdk.App();
+  // WHEN
+  const stack = new LayerPublisher.LayerPublisherStack(app, 'MyTestStack', {
+    layerName: 'AWSLambdaPowertoolsTypeScript',
+    powerToolsPackageVersion: '1.0.1',
```

Comment:
Same as one of the comments above, does this version stay in sync with the published one?

Reply:
no just unit test stuff